echopop/live/live_data_loading.py

```diff
@@ -0,0 +1,928 @@
+    # ---- Extract the expected file name ID's
+    biology_file_ids = file_settings["file_name_formats"]
+    # ---- Extract all of the file ids
+    biology_config_ids = list(biology_file_ids.keys())
+    # ---- Initialize the dictionary that will define this key in the `input` attribute
+    biology_output = {f"{key}_df": pd.DataFrame() for key in biology_config_ids}
+    # # ---- Create filepath object
+    if "data_root_dir" in file_configuration:
+        # directory_path = Path(file_configuration["data_root_dir"]) / file_settings["directory"]
+        directory_path = "/".join([file_configuration["data_root_dir"], file_settings["directory"]])
+    else:
+        directory_path = file_settings["directory"]
+
+    # Add SQL file to dict
+    # file_configuration["database"]["biology"] = (
+    #     Path(file_configuration["data_root_dir"]) / "database" / file_settings["database_name"]
+    # )
+    file_configuration["database"]["biology"] = (
+        # Path(file_configuration["database_directory"]) / file_settings["database_name"]
+        "/".join([file_configuration["database_directory"], file_settings["database_name"]])
+    )
+
+    # Iterate through the different biology datasets and read them in
+    for dataset in list(biology_file_ids.keys()):
+        # ---- Get dataset-specific file lists
+        dataset_files = filter_filenames(
+            directory_path, biology_file_ids[dataset], biology_files, file_settings["extension"]
+        )
+        # ---- If there are dataset files available
+        if dataset_files:
+            # ---- Read in validated biology data
+            dataframe_list = [
+                read_biology_csv(
+                    file,
+                    file_settings["file_name_formats"][dataset],
+                    biology_config_map[dataset],
+                    pandas_kwargs,
+                )
+                for file in dataset_files
+            ]
+            # ---- Concatenate the dataset
+            dataframe_combined = pd.concat(dataframe_list, ignore_index=True)
+            # ---- Lower-case sex
+            if "sex" in dataframe_combined.columns:
+                dataframe_combined["sex"] = dataframe_combined["sex"].str.lower()
+            # ---- Lower-case trawl partition type
+            if "trawl_partition" in dataframe_combined.columns:
+                dataframe_combined["trawl_partition"] = dataframe_combined[
+                    "trawl_partition"
+                ].str.lower()
+            # ---- Reformat datetime column
+            if "datetime" in dataframe_combined.columns:
+                dataframe_combined["datetime"] = convert_datetime(dataframe_combined["datetime"])
+            # ---- Add to the data dictionary
+            biology_output[f"{dataset}_df"] = dataframe_combined
+
+    # Return the output
+    return biology_output
+
+
+def read_acoustic_zarr(file: Path, config_map: dict, xarray_kwargs: dict = {}) -> tuple:
```

Comment:
Suggest changing this to `nasc_zarr_to_df` so that it is clear what it does. Also to avoid the similarity with `read_acoustics_files` that has acousticS and here the function name does not have an S...